src/validationinterface.h

```diff
@@ -131,9 +131,18 @@ class CValidationInterface {
      * Called on a background thread.
      */
     virtual void TransactionRemovedFromMempool(const CTransactionRef& tx, MemPoolRemovalReason reason, uint64_t mempool_sequence) {}
+    /* Notifies listeners of transactions removed from the mempool as
+     * as a result of new block being connected.
+     * MempoolTransactionsRemovedForConnectedBlock will be fired before BlockConnected.
+     *
+     * Called on a background thread.
+     */
+    virtual void MempoolTransactionsRemovedForConnectedBlock(const std::vector<CTransactionRef>& txs_removed_for_block, unsigned int nBlockHeight) {}
```

Comment:
This function is newly added in 4986edb99f8aa73f72e87f3bdc09387c3e516197 and then changed in 863d46751900a0c69c4f7fbd92d4d8a651b2cc7b

Reply:
Fixed